script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -208,7 +208,7 @@ But I can't afford to lose. I will not fall here.
 [sha:f682342cd3bf7c0d3c1b208f376bc728d813f109]{
 -- Page 116, Offset 35589.
 -- ノエルの手には一本の凶器が握られている。
-Noel holds one of her instruments of murder in her hand.
+Noel holds a single of her instruments of murder in her hand.
```

Comment:
Either revert or fix

Reply:
Probably rewrite, because of the line below

Reply:
Noel holds an instrument of murder in her hand.